README.md

```diff
@@ -1 +1,39 @@
-# hello-microverse-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Pull Request should have a descriptive title telling you what changes are inside, kindly add a more descriptive title to your PR

Reply:
@Ismailco I can't figure out how to change the title for the pull request if it's already open, should I close it and open a new one?